src/utils/VoiceAnalyser.js

```diff
@@ -121,6 +157,87 @@ function VoiceAnalyser(props) {
     }
   };
 
+  const getResponseText = async (audioBlob) => {
```

Comment:
Later need to move this in different component - should be configurable. the sample rate might be different for different asr services